docs/source/search-syntax.rst

```diff
@@ -80,21 +83,50 @@ For example:
 Run Attributes
 ~~~~~~~~~~~~~~
 
-The search syntax supports searching runs using two attributes: ``status`` and ``artifact_uri``. Both attributes have string values. Other fields in :py:class:`mlflow.entities.RunInfo` are :ref:`system_tags` that are searchable using the UI and the API. The search returns an error if you use other attribute names in the filter string. 
+You can search using two run attributes contained in :py:class:`mlflow.entities.RunInfo`: ``status`` and ``artifact_uri``. Both attributes have string values. 
+
+Other fields in ``mlflow.entities.RunInfo`` are :ref:`MLflow tags <system_tags>` that are searchable as ``tag`` identifiers (see :ref:`mlflow_tags` below); search returns an error if you use these names in the ``attributes`` identifier. 
```

Comment:
@stbof So I think we might want to rephrase this because technically MLflow tags aren't in `mlflow.entities.RunInfo`. Some metadata in `RunInfo` can be searched, specifically `status` and `artifact_uri` (as described above. However, MLflow tags are separate from the `RunInfo` entity because they would show up as tags in the `RunData` entity.

Reply:
Hmm, I was just rephrasing what already existed, namely "Other fields in :py:class:`mlflow.entities.RunInfo` are :ref:`system_tags` that are searchable using the UI and the API."

@mparkhe what should we be saying here?

Reply:
Agreed. Only the first part of the sentence makes sense. If we want to talk about errors we can say something like

```
Search will return an error when using :py:class:mlflow.entities.RunInfo that are not :ref:system_tags
```

The `note::`` below this paragraph does provide more context should the reader be tempted.

Reply:
@mparkhe But I think it's confusing to say that some `RunInfo` fields are also MLflow system tags. There's no overlap in the RunInfo fields and system tags. The closest between the two are RunInfo's `user_id` and `run_id` with system tags `mlflow.user` and `mlflow.runName`. But The latter are not actual identifiers. 

I think it makes more sense to say:
```
...``status`` and ``artifact_uri``. Both attributes have string values. Other fields in ``mlflow.entities.RunInfo`` are not searchable.
```

Then in the `MLflow Tags` section, we can add a reference to the `System Tags` page.
@stbof what do you think?

Reply:
I agree that we should not talk at all about System Tags in the Run Attributes section. We should reference them only in the MLflow Tags section.